umbraco/Umbraco.CMS.Backoffice
# Bugfix: Add getter for pickerContext in UmbInputEntityElement

## Description

To support Lit 3 we were missing a getter for the pickerContext field. It won't make any difference at the moment other than removing a warning when running our tests.

## How to test
* Run the tests before this fix and see a lot of warnings about the missing getter
* Run the tests after this fix and the warning will be gone.

## Types of changes
